AISKU/Tests/Unit/src/GlobalTestHooks.Test.ts

```diff
@@ -0,0 +1,13 @@
+import { Assert } from "@microsoft/ai-test-framework";
+import { _testHookMaxUnloadHooksCb } from "@microsoft/applicationinsights-core-js";
+import { dumpObj } from "@nevware21/ts-utils";
+
+export class GlobalTestHooks {
+
+    public registerTests() {
```

Comment:
This is using a "hack" of our current test infrastructure to "set" some global configuration that will affect "ALL" tests that follow the initialization. (which is why this was added as the "first" item of every tests bundle.

Of note, is that IF any test also changes / sets / calls the test hook it will "replace" and invalidate this setting (as this is currently a single value, not a list of callbacks)

Reply:
They also needed to be created / added here as the "common" test infrastructure does not have access to the main "Application Insights" code to be able to add code specific "hooks"